tests/foreman/cli/test_setting.py

```diff
@@ -33,315 +34,352 @@
+        1. Execute "settings" command with "set" as sub-command
+        with any string
 
-    @tier2
-    def test_positive_update_hostname_default_prefix(self):
-        """Update the default set prefix of hostname_prefix setting
+    :expectedresults: Parameter is updated successfully
 
-        :id: a6e46e53-6273-406a-8009-f184d9551d66
+    :parametrized: yes
+    """
+    login_text_value = [login_text for login_text in valid_data_list()][0]
+    Settings.set({'name': "login_text", 'value': login_text_value})
+    login_text = Settings.list({'search': 'name=login_text'})[0]
+    assert login_text["value"] == login_text_value
+
+
+@tier2
+@pytest.mark.parametrize('setting_update', ['login_text'], indirect=True)
+def test_positive_update_login_page_footer_text_without_value(setting_update):
+    """Updates parameter "login_text" without any string (empty value)
+
+    :id: 01ce95de-2994-42b6-b9f8-f7882981fb69
+
+    :steps:
+
+        1. Execute "settings" command with "set" as sub-command
+        without any string(empty value) in value parameter
+
+    :expectedresults: Message on login screen should be removed
+
+    :parametrized: yes
+    """
+    Settings.set({'name': "login_text", 'value': ""})
+    login_text = Settings.list({'search': 'name=login_text'})[0]
+    assert login_text['value'] == ''
 
-        :expectedresults: Default set prefix should be updated with new value
-        """
-        hostname_prefix_value = gen_string('alpha')
-        Settings.set({'name': "discovery_prefix", 'value': hostname_prefix_value})
-        discovery_prefix = Settings.list({'search': 'name=discovery_prefix'})[0]
-        self.assertEqual(hostname_prefix_value, discovery_prefix['value'])
 
-    @pytest.mark.stubbed
-    @tier2
-    def test_positive_update_hostname_default_facts(self):
-        """Update the default set fact of hostname_facts setting with list of
-        facts like: bios_vendor,uuid
+@tier2
+@pytest.mark.parametrize('setting_update', ['login_text'], indirect=True)
+def test_positive_update_login_page_footer_text_with_long_string(setting_update):
+    """Attempt to update parameter "Login_page_footer_text"
+        with long length string under General tab
 
-        :id: 1042c5e2-ee4d-4eaf-a0b2-86c000a79dfb
+    :id: 87ef6b19-fdc5-4541-aba8-e730f1a3caa7
 
-        :expectedresults: Default set fact should be updated with facts list.
+    :steps:
+        1. Execute "settings" command with "set" as sub-command
+        with long length string
 
-        :CaseAutomation: notautomated
-        """
+    :expectedresults: Parameter is updated
 
-    @pytest.mark.stubbed
-    @tier2
-    def test_negative_discover_host_with_invalid_prefix(self):
-        """Update the hostname_prefix with invalid string like
-        -mac, 1mac or ^%$
+    :parametrized: yes
 
-        :id: 73c5da42-28c8-4be7-8fb1-f505fefd6665
+    :CaseImportance: Low
+    """
+    login_text_value = [login_text for login_text in generate_strings_list(1000)][0]
```

Comment:
Same comment as in other PR - I don't like that we are just throwing away coverage without any discussion or consideration. Perhaps we don't need coverage for everything in every run, but we could randomize value in test to maintain coverage over longer periods of time.

Reply:
I agree with your statement but generate_strings_list(1000) returns the list of a random value.

```
[login_text for login_text in generate_strings_list(1000)][0]

'𘖂碥ᴚ𐃃焻颃貘襮衧㲲ࡐ䢵䢐纚鑭䍍弰ｮ阻텵ん淋𗦑솉充䜒墡萩𒔂ȿ𗛀蛦惆乾ᖛ𗞯䂮𞤙燧ᙹ컜ଠ쇧趙𓄠䈻𝚒却𝛣柽哶꿫ꑋ𖩏𓀍퉽읛擒ꔌ𐙖訣ꍖ...誚璹
```

Reply:
Only if your robottelo.properties has `run_one_datapoint=true`. It is false by default and explicitly on CI.

Reply:
Yes, you are right, I have added the suggested changes.